packages/run-it/src/utils/RunItSDK.ts

```diff
@@ -137,3 +137,7 @@ export const sdkNeedsAuth = (sdk: IAPIMethods | undefined) => {
   const configIsNeeded = sdkNeedsConfig(sdk)
   return configIsNeeded && !sdk.authSession.isAuthenticated()
 }
+
+export enum StoreConstants {
+  LOCALSTORAGE_SETTINGS_KEY = 'settings',
+}
```

Comment:
Moved this out of extension provider into RunIt which has a "base" redux store that can be used with API Explorer also